src/lib/users/ssh_authorized_keys_file.rb

```diff
@@ -71,6 +71,10 @@ def keys
       # @return [Boolean] +true+ if the key was added; +false+ otherwise
       def add_key(key)
         new_key = key.strip
+
+        # Ignores comments or empty lines given as key
+        return if new_key.empty? || new_key.start_with?("#")
```

Comment:
what about line with spaces?

Reply:
Note that _new\_key_  is the result of `key.strip`. So, "     "  => "" => .empty? == true